devito/ir/iet/visitors.py

```diff
@@ -678,18 +678,19 @@ class IsPerfectIteration(Visitor):
     def visit_object(self, o, **kwargs):
         return False
 
-    def visit_tuple(self, o, **kwargs):
-        return all(self._visit(i, **kwargs) for i in o)
+    def visit_tuple(self, o, found=False, nomore=False):
```

Comment:
nitpicking: TBH nomore is a bit vague. Although I get its use....just seems  'ugly'... ;-)

Reply:
any suggestion? I just reused what was there before

Reply:
maybe `final`, `last`, `ultimate`, or `ending` ?

Reply:
I think `nomore` conveys the idea to be fair

all your suggestions don't make much sense to me